src/dracula.yml

```diff
@@ -26,21 +26,38 @@ scheme:
   FindMatch: &FindMatch         '#f8f8f220'
   
   # Opaque Colors
-  PURPLE30: &PURPLE30  '#bd93f930'
-  YELLOW50: &YELLOW50  '#f1fa8c50'
+  PURPLE30: &PURPLE30 '#bd93f930'
+  PURPLE90: &PURPLE90 '#bd93f990'
+  PURPLEAA: &PURPLEAA '#bd93f9AA'
+  YELLOW30: &YELLOW30 '#f1fa8c30'
+  YELLOW50: &YELLOW50 '#f1fa8c50'
```

Comment:
Try to avoid leaving the main color palate if at all possible (that's not to say that the cases that you did didn't require it, but just throwing that out there!). Straying from main scheme should be a last resort.